src/org/zaproxy/zap/extension/tokengen/TokenGenMessagesTableModel.java

```diff
@@ -117,6 +117,15 @@ public Object getValueAt(int rowIndex, int columnIndex) {
         }
     }
 
+    @Override
+    public boolean isCellEditable(int rowIndex, int columnIndex) {
```

Comment:
How about allow cell selection instead? One could use the usual keyboard shortcut to copy individual cell values (it also works with non-String ones).